pgsync/sync.py

```diff
@@ -418,10 +430,20 @@ def should_skip_event(self, event: dict) -> bool:
                 that are skippable as per settings.SKIP_NULL_KEYS
         """
 
-        logger.debug(f"validating event: {event}")
-
-        if (event['schema'], event['table']) not in self.valid_tables:
-            logger.debug(f"skipping event for {event['schema']}.{event['table']} as it's not in the configured pgsync JSON schema")
+        logger.debug(f"should_skip_event: validating event: {event}")
+
+        if (event['schema'], event['table']) in self._gist_schema_attributes:
+            if CHANGED_FIELD_NAME in event and event[CHANGED_FIELD_NAME] is not None:
+                skip_status = True
+                for column in event[CHANGED_FIELD_NAME]:
+                    if column in self._gist_schema_attributes[(event['schema'], event['table'])]:
+                        skip_status = False
+                        break
```

Comment:
```python
=> fields = {('mc_hammer', 'project'): {'name', 'id', 'p2_project_id'}, ('mc_hammer', 'area'): {'name', 'id', 'market'}}

=> any(field in ['name', 'foo', 'bar'] for field in fields[('mc_hammer', 'area')])
True

=> any(field in ['name', 'foo', 'bar'] for field in fields[('mc_hammer', 'area')])
False
```

Reply:
my version is more readable for me, and I don't really understand how you got different results for the **any(...)** while executed, correct result is True